app/controller.js

```diff
@@ -0,0 +1,15 @@
+const {selectAllTopics} = require("./model.js")
+
+
+
+
+
+exports.getTopics = (req,res,next) =>{
+
+
+    selectAllTopics().then(({rows})=>{
+  
+  
+        return res.status(200).send(rows)
```

Comment:
any data from a controller needs to be sent back on a meaningful key, ie. .send({topics: rows})